php-7.3-apache/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM php:7.3-apache
+
+MAINTAINER Cid Lopes "alannettto@gmail.com"
+
+ENV TERM=xterm
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+RUN apt-get update && \
```

Comment:
Can you reformat to be in separate lines? One dependency per line. I normally just run this multiple times and then flatten the layers.

`RUN apt-get -y -qq install <dep>`

It is more readable but I'm not sure what are the other drawbacks.